tools/dependencies.toml

```diff
@@ -3,6 +3,11 @@ version="0.16.0"
 tarpath="https://github.com/fluxcd/flux2/releases/download/v${version}/flux_${version}_${goos}_${goarch}.tar.gz"
 bindir="../pkg/flux/bin"
 
+[fluxchecksums]
```

Comment:
I think it would be cleaner to add the checksum txt file to the [flux] section instead of creating a new download here.  It will require less change (version number) as we update the flux versions.